bundles/org.openhab.binding.hive/src/main/java/org/openhab/binding/hive/internal/client/dto/FeaturesDto.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hive.internal.client.dto;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ *
+ *
+ * @author Ross Brown - Initial contribution
+ */
+@NonNullByDefault
+public final class FeaturesDto {
+    public @Nullable AutoBoostV1FeatureDto autoboost_v1;
```

Comment:
Please use camelcase for field names.

Reply:
I did this because the API returns them in snake case.
I know I could use annotations to tell GSON about this and use camel case field names but I didn't want to use library-specific annotations in my DTOs if I could avoid it.